projects/cardiovascular-patient-app/src/app/report-bp/report-bp.component.spec.ts

```diff
@@ -109,6 +103,36 @@ describe('ReportBPComponent', () => {
     await systolicInputHarness.setValue('252');
     expect(systolicFormField?.errors?.['max']).toBeTruthy();
   });
+});
+
+describe('submit test', () => {
+  let component: ReportBPComponent;
+  let fixture: ComponentFixture<ReportBPComponent>;
+  let loader: HarnessLoader;
+  let fhirDataService: jasmine.SpyObj<FhirDataService>;
+  let matSnackBarSpy: jasmine.SpyObj<MatSnackBar>;
```

Comment:
I don't think this submit test needs to be in a different describe block. The tests above should be using this same configuration.

Reply:
Okay, @semicolin  I will move this submit the test to the upper describe block.